test/run-in-travis.sh

```diff
@@ -21,8 +21,25 @@ regression_tests() {
 	sh /usr/share/crmsh/tests/regression.sh
 }
 
-unit_tests
-rc_unittest=$?
-configure
-make_install
-regression_tests && exit $rc_unittest
+bootstrap_tests() {
+	echo "** Bootstrap process tests using python-behave"
+        behave --no-logcapture --tags "@bootstrap" --tags "~@wip" /usr/share/crmsh/tests/features/bootstrap_$1.feature
```

Comment:
I would change that, and pass as argument the howl `.feature` file name. It would be easier to expand in the future and we can have more flexibility selecting `feature` file names

Reply:
Changed like this:
```
bootstrap_tests() {
        SUFFIX="${1:-*}"
        echo "** Bootstrap process tests using python-behave"
        behave --no-logcapture --tags "@bootstrap" --tags "~@wip" /usr/share/crmsh/tests/features/bootstrap_$SUFFIX.feature
}
```